src/ItemsStorage.php

```diff
@@ -70,6 +71,16 @@ public function get(string $name): ?Item
         return empty($item) ? null : $this->populateItem($item);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function exists(string $name): bool
+    {
+        $item = $this->database->select([new Fragment('1')])->from($this->tableName)->where(['name' => $name])->run()->fetch();
```

Comment:
I think you can use `count()` here.
But the better case may be adding for `limit(1)` in all similar queries.